master/docs/manual/configuration/builders.rst

```diff
@@ -211,4 +211,15 @@ The virtual builder metadata is configured with the following properties:
 
 * ``virtual_builder_tags``: The tags for the virtual builder.
 
+You can also use virtual builders with :bb:sched:`SingleBranchScheduler`. For example::
```

Comment:
`You can also use virtual builders with :bb:sched:`SingleBranchScheduler` for example if you want to automatically build all branches in your project without having to manually create a new builder each time one is added.`

Something like that might make it clearer as to how to solve this common use-case.